webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueCompilationCache.scala

```diff
@@ -41,9 +41,32 @@ import com.normation.errors.IOResult
 import com.normation.inventory.domain.Version
 import com.normation.rudder.batch.UpdateCompilationStatus
 import com.normation.rudder.domain.logger.ConfigurationStatusLoggerPure
+import com.normation.rudder.ncf.CompilationResult.Success
 import net.liftweb.common.SimpleActor
 import zio.*
-import zio.NonEmptyChunk
+
```

Comment:
more data structure to handle the different steps of the process. This one is the one after gathering information